phmi/management/commands/load_justification_details.py

```diff
@@ -1,78 +1,94 @@
 import csv
+import os
 
 from django.core.management.base import BaseCommand
 from django.db import transaction
 
-from phmi import models
+from ...models import LegalJustification, OrgType
 
 ORG_TYPES = ["CCG", "NHS Trust", "Local Authority", "NHS England"]
 
 
 class Command(BaseCommand):
-    help = "Load in some activities"
-
-    def get_justifications(self, justification_activity, org_type_name):
-        """
-        There are some differences in terms of the prefix
-        (e.g. commsissioner duties vs duties) so we
-        search by the suffix. Then do a match.
-        """
-        org_type = models.OrgType.objects.get(name=org_type_name)
-        prefix, justification_suffix = justification_activity.split(":", 1)
-        legal_justification = models.LegalJustification.objects.filter(
-            org_type=org_type
-        )
-        legal_justification = legal_justification.filter(
-            name__endswith=justification_suffix
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--path", default="data/csvs/statutes", help="Path to load Statutes from"
         )
 
-        result = []
-        for i in legal_justification:
-            if prefix.lower() in i.name.split(":")[0].lower():
-                result.append(i)
+    # def get_justifications(self, justification_activity, org_type_name):
```

Comment:
Do we really want to comment all the things? Can this just be deleted?

Reply:
Woops, didn't mean to leave those bits in, sorry about that!